libs/blocks/modal/modal.js

```diff
@@ -49,6 +49,18 @@ export function sendAnalytics(event) {
   }
 }
 
+function focusAfterModalClose(modal) {
+  const { id } = modal;
+  const isGeoPopup = id === 'locale-modal-v2';
```

Comment:
We should use a constant for `locale-modal-v2`.
```suggestion
  const LOCALE_MODAL_ID = 'locale-modal-v2';
  ...
  const isGeoPopup = id === LOCALE_MODAL_ID;
```